HotelManagement/src/core/Booking.java

```diff
@@ -0,0 +1,80 @@
+package core;
+
+import java.time.LocalDate;
+
+public class Booking {
```

Comment:
JavaDoc on class level with brief explanation about the class.

Reply:
Done